projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -4424,4 +4425,21 @@ private void writeJsonTopology() {
       throw new BatfishException("Failed to synthesize JSON topology", e);
     }
   }
+
+  public void checkTopology(Map<String, Configuration> configurations, Topology topology) {
+    for (Edge edge : topology.getEdges()) {
+      if (!configurations.containsKey(edge.getNode1())
+          || !configurations.containsKey(edge.getNode2())) {
+        throw new BatfishException("ERROR: topology contains a non-existent node\n");
```

Comment:
Does the format of this message match the other exceptions we throw? E.g., Line 4425 does not seem to include an `ERROR:` prefix.